stamp_generation/stamp_templates/stamp_template.html.jinja

```diff
@@ -201,8 +255,24 @@
         publisher="{{publisher_domain}}"
         publisher-logo-src="{{logo_url}}"
         poster-portrait-src="abc.jpg">
+        <amp-story-page id="page-1">
+            <amp-story-grid-layer template="fill" >
+                    <amp-img object-fit="contain" layout="fill" class="title-page"
+                         src="https://www.google.com/s2/favicons?sz=256&domain_url={{publisher_domain}}">
```

Comment:
Can you check for other domains if this size is always available?

Apart from this url, there are other ways to get URL, can we try that as well.

Reply:
No, this size is not always available so logo will be blurred in that case. And yes we can try some other API. I will look into it.

Reply:
> And yes we can try some other API. I will look into it.

No, not API. If I remember correctly, one stack overflow answer mentioned about getting image from html. It is the same answer where we figured out this API: https://www.google.com/s2/favicons

Reply:
Okay! I will look into ways to extract the logo url from original page.

Reply:
Searching around I also found https://clearbit.com/logo - only thing is they require attribution. If you want to use that, probably the last page should have 'logo provided by clearbit.com'

Not all websites seem to have 256 size favicon (example, this looks bad: https://www.google.com/s2/favicons?sz=256&domain_url=arre.co.in).

On the flip side,  a simple color background color for now and later we could add an image from the article (possibly the largest one) is also fine. 

(The layout itself looks really good, lets try to get that in)

Reply:
+1 layout looks awesome.

Reply:
I was talking about this [answer](https://stackoverflow.com/a/5119097).

Reply:
I will look into the possible options to get good size url for background (Including [link1](https://stackoverflow.com/a/5119097) and [link2](https://clearbit.com/logo) ).

Reply:
And if it appears that a particular size logo is not there for given domain then we can go for **plain background**/**image in background** option.

Reply:
Using clearbit API. Thanks @droidkid for suggestion.